services/painting-surface-defect-detection-model-service/app/routers/predict.py

```
@@ -0,0 +1,198 @@
+
+# 로깅 설정
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+# 전역 서비스 인스턴스 (main.py에서 설정됨)
+detection_service: Optional[PaintingSurfaceDefectDetectionService] = None
+
+
+from pydantic import BaseModel, validator, Field
```

Ruff (0.12.2)

Redefinition of unused `BaseModel` from line 2

Remove definition: `BaseModel`

(F811)